aptos-move/framework/aptos-token-objects/sources/aptos_token.move

```diff
@@ -289,22 +289,22 @@ module aptos_token_objects::aptos_token {
 
     // Token mutators
 
-    inline fun authorized_borrow<T: key>(token: &Object<T>, creator: address): &AptosToken {
```

Comment:
This function is private though so there should be no risk with another module calling it?

Reply:
Right, there is no security concern here. 

However, `authorized_borrow` should be a nice pattern that many other custom tokens will follow to use potentially (e.g., `authorized_worker_borrow` to authorize the worker account and borrow the reference of token in some token module). I thought it would be great to set a good example in this standard module because other people could refer this pattern.

And, I think the right way to "authorize" is to check `&signer` instead of `address` in principle.